### PR TITLE
Use SolaraViz instead of JupyterViz in examples

### PR DESCRIPTION
**Pull request by EwoutH**
This updates all the examples to use the renamed and stabilized SolaraViz.

For reference:
- https://github.com/projectmesa/mesa/pull/2090
- https://github.com/projectmesa/mesa/pull/2188

### CODE REVIEW

**Review by rht — APPROVED** [review]

**Comment by EwoutH** [comment]
Thanks for merging!

File: examples/schelling_experimental/app.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from mesa.visualization.jupyter_viz import JupyterViz, Slider, make_text
+from mesa.visualization.solara_viz.py import Slider, SolaraViz, make_text
```
Comment (rht) [inline examples/schelling_experimental/app.py:1]: This line doesn't work.

Comment (EwoutH) [reply examples/schelling_experimental/app.py:1]: Do you think because of the re-order? Ruff mandated that.

Comment (rht) [reply examples/schelling_experimental/app.py:1]: Yes, removing .py fixed it.

Comment (EwoutH) [reply examples/schelling_experimental/app.py:1]: Fixed in 274a1bcdce13d86f992eed085093f8954400ad98

**Pull request merged**